test/lsp.py

```diff
@@ -480,11 +509,48 @@ def test_textDocument_didOpen_with_relative_import(self, solc: JsonRpcProcess) -
         report = published_diagnostics[1]
         self.expect_equal(report['uri'], self.get_test_file_uri('lib'), "Correct file URI")
         self.expect_equal(len(report['diagnostics']), 1, "one diagnostic")
-        self.expect_diagnostic(report['diagnostics'][0], code=2072, lineNo=31, startEndColumns=(8, 19))
+        marker = self.getFileMarkers("lib")["@diagnostics"]
+        self.expect_diagnostic(report['diagnostics'][0], code=2072, marker=marker)
+
+    def getFileMarkers(self, test_name: str, verbose=False):
```

Comment:
`get_file_markers`? Do we have coding style checks for python?

Reply:
The style is so all over the place I wasn't sure what to follow. I am fine with either, just pick one :D